Ansible.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ansible retry files
+*.retry
+
+# Test driven development on Ansible with Molecule, Testinfra and pytest
+.cache/
+.idea/
```

Comment:
Some of these rules are covered by other templates we maintain - `Global/JetBrains.gitignore` and `Python.gitignore` come to mind. Are we able to distill these rules down to just what Ansible needs?